parquet-plugins/parquet-plugins-benchmarks/src/main/java/org/apache/parquet/plugins/benchmarks/ByteBitPackingVectorBenchmarks.java

```diff
@@ -38,48 +40,60 @@
  */
 
 @State(Scope.Benchmark)
-@BenchmarkMode(Mode.AverageTime)
+@BenchmarkMode(Mode.Throughput)
 @Warmup(iterations = 1, batchSize = 100000)
 @Measurement(iterations = 1, batchSize = 100000)
-@OutputTimeUnit(TimeUnit.MILLISECONDS)
+@OutputTimeUnit(TimeUnit.SECONDS)
+@Fork(jvmArgsPrepend = {"--add-modules=jdk.incubator.vector"})
```

Comment:
Will this flag change in the future jdk release?

Reply:
A transition of VectorAPI from an incubator module will request changes in both tests and vectorized packers ([PARQUET-2159](https://issues.apache.org/jira/browse/PARQUET-2159)) in future.